rpi3-convert-stage-5.sh

```diff
@@ -16,26 +16,60 @@
 
 application_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 output_dir=${application_dir}/output
+uboot_dir=${output_dir}/uboot-mender
 bin_dir_pi=${output_dir}/bin/raspberrypi
 sdimg_base_dir=$output_dir/sdimg
 
 echo "Running: $(basename $0)"
 declare -a sdimgmappings
 declare -a sdimg_partitions=("boot" "primary" "secondary" "data")
 
+# Takes following arguments:
+#
+#  $1 - ARM toolchain
 build_uboot_files() {
+  local CROSS_COMPILE=${1}-
+  local ARCH=arm
+  local revision="mender-rpi-2017.09"
```

Comment:
I would use a fixed revision here instead of branch.

Reply:
Do you mean that I should add a tag to mendersoftware/uboot-mender repository to the latest commit (and checkout tagged version), or simply checkout specific commit?

Reply:
Specific commit is enough I would say.